tachyon/crypto/random/rng_type.h

```diff
@@ -0,0 +1,47 @@
+#ifndef TACHYON_CRYPTO_RANDOM_RNG_TYPE_H_
+#define TACHYON_CRYPTO_RANDOM_RNG_TYPE_H_
+
+#include <stdint.h>
+
+#include <string>
+
+#include "absl/strings/substitute.h"
+
+#include "tachyon/base/flag/flag_value_traits.h"
+
+namespace tachyon {
+namespace crypto {
+
+// THE ORDER OF ELEMENTS ARE IMPORTANT!! DO NOT CHANGE!
+// This order matches with the constants of c APIs.
+// Pleas refer to tachyon/c/crypto/random.h for details.
+enum class RngType : uint8_t {
```

Comment:
```suggestion
enum class RNGType : uint8_t {
```
Optional comment. You can resolve it yourself if you think it's not needed.